subsys/bluetooth/Kconfig

```
@@ -181,6 +181,13 @@ config BT_PATH_LOSS_MONITORING
 	  Enable support for LE Path Loss Monitoring feature that is defined in the
 	  Bluetooth Core specification, Version 5.4 | Vol 6, Part B, Section 4.6.32.
 
+config BT_SUBRATING
+	bool "LE Connection Subrating"
```

New APIs should start out experimental.

```suggestion
	bool "LE Connection Subrating"
	select EXPERIMENTAL
```

See https://docs.zephyrproject.org/latest/develop/api/api_lifecycle.html for more info.

---

Thanks, will push together with any further review changes